### DISCUSSION TITLE
question about the "sel" and "rcut" parameter settings

### DISCUSSION THREAD

**Discussion by xiaoqi-neu1** [deepmodeling/deepmd-kit#3090] [Q&A]
Is the "sel" parameter setting important? How can one determine the value of the "sel" parameter? If dealing with H2O, how can one obtain the "sel" parameter? Also, when setting "rcut" to 10.0, there is always insufficient memory. For my dataset, 216H2O, the "rcut" parameter should be set to 10.0, but setting it to 10.0 results in insufficient memory. What should be done in such a case?

**Comment by njzjz** [comment]
See #108
